modules/maven-archetype/maven-archetype-springapp/src/main/resources/archetype-resources/src/main/java/com/helloapp/controllers/HelloWorldController.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.helloapp.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.bind.annotation.PathVariable;
+
+/**
+ * Spring Controller Class
+ */
+
+@Controller
+public class HelloWorldController {
+
+    /** stores the string to be displayed specific to the methods being invoked */
+    String msg="";
+    /**
+     * Method called when the request comes in the form of "/"
+     */
+    @RequestMapping(value = "/")
+    public ModelAndView indexPageCallback() {
+
+        //creating a new ModelAndView object and indicating the location of the view page
+        ModelAndView mav = new ModelAndView("index");
+
+       //the message to be passed
+        msg = "Hello World!!";
```

Comment:
Since this PR had several conflicts i was asked to create a new one by the AF team and the new PR is here https://github.com/wso2/product-af/pull/649. Suggested change was already made there.

Thanks